test/testdata/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -1776,6 +1776,14 @@ items:
     - ""
     attributeRestrictions: null
     resources:
+    - nodes
```

Comment:
@smarterclayton didn't you want this role bindable by project-admins?  This will prevent that from being possible.

As for our system router, it doesn't worry overly much, but for per-project routers, this makes it impossible to create those.